### PR TITLE
Reimplemented insertProduct-function

### PR DESCRIPTION
**Pull request by MilenaZurmuehl**
Please review the code and merge it to the rest!

### CODE REVIEW

**Comment by vincentdelitz** [comment]
@MilenaZurmuehl can you please add the correct link in the index.htm, so that I can test the functionality. And in your current branch I see 3 servlets for insertProduct, could you then please delete the ones from this branch that are not needed.

**Comment by MilenaZurmuehl** [comment]
@vincentdelitz the link is actually already in the htm-file (but only there, so from other pages it will not work!)! So it should also be in the uploaded version! And regarding the Servlets: these are no versions of the same servlet but different ones which we all need!! I can rename them if you think it is confusing, so it get's more obvious what they are used for!!

**Review by morvalgo — COMMENTED** [review]
Please do NOT merge it, because I am not sure whether Milena has worked with the version of the branch insertProduct2 that i commited and pushed. I think the error handling that I implemented could be lost if you now merge it. I think Vincent should check it, because he has the best knowledge with git!

**Comment by vincentdelitz** [comment]
@morvalgo @demirdagemir @brunofiu @Hermionee It works for me now, too, but Moritz functionality with the error handling won't anymore, as I had to resolve some merge conflicts. I think he needs to adapt it to the new structure in the next days. Could anyone of you still please review the code and then approve the pull request?

File: BaseProject/WebContent/index.htm [hunk 1]
```diff
@@ -27,8 +27,10 @@
                   <li class="dropdown">
                      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Employee <span class="caret"></span></a>
                      <ul class="dropdown-menu">
+
                         <li><a href="./ShiftServlet">Search Shifts</a></li>
-			<li><a href="./insertProduct">Insert new product</a></li>
+			                  <li><a href="./insertProductServlet">Insert new product</a></li>
```
Comment (demirdagemir) [inline BaseProject/WebContent/index.htm:32]: For me this doesn't work. The servlet annotation is `InsertProductServlet` but it's `insertProductServlet` here. I think it should be capital.

File: BaseProject/WebContent/successfulInsert.jsp [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                  </li>
+                  <li class="dropdown">
+                     <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Visitor <span class="caret"></span></a>
+                     <ul class="dropdown-menu">
+                        <li><a href="#">Activity Diagram</a></li>
+                        <li><a href="./stage">Search personal stages</a></li>
+                     </ul>
+                  </li>
+                  <li class="dropdown">
+                     <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Homework Example <span class="caret"></span></a>
```
Comment (demirdagemir) [inline BaseProject/WebContent/successfulInsert.jsp:45]: We should update this header to the new one

File: BaseProject/src/de/tum/in/dbpra/InsertProductServlet.java [hunk 3]
```diff
@@ -33,61 +31,44 @@ public InsertProductServlet() {
...
-		
-		Double price = 1.00;
-		try {
-			price = Double.parseDouble(request.getParameter("price"));
-		} catch (NumberFormatException e) {
-			e.printStackTrace();
-    		request.setAttribute("error", "Invalid decimal (price)");
-		}
+	protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+//		int shopID = Integer.parseInt(request.getParameter("shopID"));
```
Comment (demirdagemir) [inline BaseProject/src/de/tum/in/dbpra/InsertProductServlet.java:53]: We could delete the commented out code

File: BaseProject/WebContent/enterProduct.jsp [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+                  </li>
+                  <li class="dropdown">
+                     <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Visitor <span class="caret"></span></a>
+                     <ul class="dropdown-menu">
+                        <li><a href="#">Activity Diagram</a></li>
+                        <li><a href="./stage">Search personal stages</a></li>
+                     </ul>
+                  </li>
+                  <li class="dropdown">
+                     <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Homework Example <span class="caret"></span></a>
```
Comment (demirdagemir) [inline BaseProject/WebContent/enterProduct.jsp:45]: Same old header here

**Review by demirdagemir — CHANGES_REQUESTED** [review]
For me the link insert product didn't work. Commented inside the code. Does it work for everyone else? Also we should update the headers because almost all of them has different links.

Other than that the logic works. (Inserting new product/custom product)

**Comment by vincentdelitz** [comment]
@demirdagemir are your requested changes still valid, as you have now "outsourced" the header issues into the header.jsp?

**Review by demirdagemir — APPROVED** [review]

**Pull request merged**